### PR TITLE
[Tests-Only]Add tests containing `[` and `]` in path names

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This PR adds tests for names containing `[` and `]` in path names and run the tests in sqlite db

## Related Issue
- https://github.com/owncloud/core/issues/31394

## How Has This Been Tested?
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiProvisioning-v1/addUser.feature [hunk 1]
```diff
@@ -197,3 +197,20 @@ Feature: add user
     Then the OCS status code should be "105"
     And the HTTP status code should be "200"
     And user "brand-new-user" should not exist
+
+  @sqliteDB
+  Scenario: admin tries to create user with brackets in the username
+    Given the following users have been deleted
+      | username  |
+      | [user1]   |
+      | [ user2 ] |
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiProvisioning-v1/addUser.feature:206]: 🤔 is user deletion really needed before the test? are the users created before the test anywhere?

Comment (jasson99) [reply tests/acceptance/features/apiProvisioning-v1/addUser.feature]: NO, they aren't. We had this step in the user creation tests so I sort of followed that. I have removed this part now from my tests.

**Review by SwikritiT — COMMENTED** [review]
LGTM :+1:

**Review by swoichha — COMMENTED** [review]
lgtm

**Review by Talank — COMMENTED** [review]
LGTM

**Review by individual-it — APPROVED** [review]

**Pull request merged**